### PR TITLE
[Tests-only] run js tests on drone

### PR DESCRIPTION
**Pull request by haribhandari07**
### Description
Run js test on drone

Issue #935

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: Travis fails because of https://github.com/Graffino/browser-update dependency tag problems.

See my comments in https://github.com/Graffino/browser-update/pull/1

File: .drone.starlark [hunk 1]
```diff
@@ -477,6 +477,11 @@ def javascript():
 				'pull': 'always',
 				'environment': params['extraEnvironment'],
 				'commands': params['extraCommandsBeforeTestRun'] + [
+					'apt update -y',
+					'apt-get install xvfb libxrender1 libxtst6 libxi6 -y',
+					'/usr/bin/Xvfb :99 &',
+					'export DISPLAY=:99.0',
+					'apt-get install firefox -y',
```
Comment (phil-davis) [inline .drone.starlark]: These commands should be able to go in the `javascript` section of `config` up the top, see an example in `files_antivirus` `.drone.star` (where `extraCommandsBeforeTestRun` is used for `phpunit`)

Then we do not have different starlark code in this repo.

**Review by phil-davis — COMMENTED** [review]
And we will have to wait to see what happens for the https://github.com/Graffino/browser-update upstream problem before we can get a Travis run to make sure Travis still passes.

**Comment by zenopopovici** [comment]
@phil-davis  Change has been reverted for https://github.com/Graffino/browser-update. Sorry for the trouble.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@HariBhandari07 now we should switch from `.drone.yml` to `.drone.star`

When you are ready to make a PR for that, let me know and I will change the drone server settings to expect `.drone.star`

**Pull request merged**